### PR TITLE
Update Vue cli + Typescript - In preparation for Pinia upgrade

### PR DESCRIPTION
**Pull request by seeker25**
*Description of changes:*
Update Vue cli + Typescript, this puts us in a good position to get Pinia working. 

Due to the weird way we are using Vuex stores - might require a bunch of re-testing.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/jest.config.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ module.exports = {
   snapshotSerializers: [
     'jest-serializer-vue'
   ],
-  'setupFiles': ['jest-localstorage-mock'],
+  setupFiles: ['jest-localstorage-mock', 'dotenv/config'],
```
Comment (seeker25) [inline auth-web/jest.config.js:22]: We need this, so the runner in VSCODE doesn't fail loading environment variables

File: auth-web/src/components/auth/create-account/AccountCreateBasic.vue [hunk 2]
```diff
@@ -136,7 +135,7 @@ export default class AccountCreateBasic extends Mixins(Steppable) {
   }
 
   private get enablePaymentMethodSelectorStep (): boolean {
-    return LaunchDarklyService.getFlag(LDFlags.PaymentTypeAccountCreation) || false
```
Comment (seeker25) [inline auth-web/src/components/auth/create-account/AccountCreateBasic.vue:139]: We don't use the LD flag anymore, I've set it to true now by default

File: auth-web/src/components/auth/staff/account-management/StaffAccountManagement.vue [hunk 3]
```diff
@@ -78,28 +78,34 @@
...
+import { KCUserProfile } from 'sbc-common-components/src/models/KCUserProfile'
+import LaunchDarklyService from 'sbc-common-components/src/services/launchdarkly.services'
+import { Component, Vue } from 'vue-property-decorator'
+import { mapActions, mapGetters, mapState } from 'vuex'
 
-import { getModule } from 'vuex-module-decorators'
 import { namespace } from 'vuex-class'
+import { getModule } from 'vuex-module-decorators'
 
+// eslint-disable-next-line no-unused-vars
```
Comment (seeker25) [inline auth-web/src/components/auth/staff/account-management/StaffAccountManagement.vue:98]: Had some typescript errors here until I added this in

File: auth-web/.browserslistrc [hunk 4]
```diff
@@ -1,3 +1,3 @@
 > 1%
 last 2 versions
-not ie <= 10
\ No newline at end of file
+not dead
```
Comment (seeker25) [inline auth-web/.browserslistrc:3]: I think this will disable IE

File: auth-web/package.json [hunk 5]
```diff
@@ -53,41 +53,48 @@
...
     "@typescript-eslint/eslint-plugin": "^5.23.0",
     "@typescript-eslint/parser": "^5.23.0",
     "@volar-plugins/vetur": "latest",
-    "@vue/cli-plugin-babel": "^4.5.17",
-    "@vue/cli-plugin-e2e-nightwatch": "^4.5.17",
-    "@vue/cli-plugin-eslint": "^4.5.17",
-    "@vue/cli-plugin-typescript": "^4.5.17",
-    "@vue/cli-plugin-unit-jest": "^4.5.17",
-    "@vue/cli-service": "^4.5.17",
+    "@vue/cli-plugin-babel": "~5.0.0",
```
Comment (seeker25) [inline auth-web/package.json:62]: This updates webpack, part of JEST, etc

**Review by pwei1018 — APPROVED** [review]

File: auth-web/src/components/auth/staff/account-management/StaffAccountManagement.vue (hunk 3 above)
Comment (seeker25) [reply auth-web/src/components/auth/staff/account-management/StaffAccountManagement.vue:98]: Will move this to seperate ENUM, that's what I had to do for `business-edit-ui`

**Comment by seeker25** [comment]
Will revisit a bit later

**Pull request closed without merging**